klippy/gcode.py

```diff
@@ -96,6 +96,7 @@ def __init__(self, printer):
         printer.register_event_handler("klippy:shutdown", self._handle_shutdown)
         printer.register_event_handler("klippy:disconnect",
                                        self._handle_disconnect)
+        self._get_status_info = {}
```

Comment:
FYI, it would be preferable to build a default response to start - that is: `self._get_status_info = {'commands': {}}`